samples/llm-failover/test.sh

```
@@ -0,0 +1,190 @@
+#   NOTE: requires embedding_provider_api_key in additional-config.toml.
+# ═══════════════════════════════════════════════════════════════════════════
+echo ""
+echo "══════════════════════════════════════════════════"
+echo " Test 2: Semantic Cache"
+echo "══════════════════════════════════════════════════"
+
+CACHE_MSG="What is the boiling point of water in Celsius?"
+
+info "Request 1 — cache miss expected ..."
+HDR1=$(mktemp)
+T1_START=$(ms); R1=$(chat_req "${CACHE_MSG}" "${HDR1}"); T1_END=$(ms)
```

Shellcheck (0.11.0)

[warning] R1 appears unused. Verify use (or export if used externally).

(SC2034)